vllm/v1/engine/async_llm.py

```diff
@@ -150,7 +151,11 @@ def _get_executor_cls(cls, vllm_config: VllmConfig) -> Type[Executor]:
         executor_class: Type[Executor]
         distributed_executor_backend = (
             vllm_config.parallel_config.distributed_executor_backend)
-        if distributed_executor_backend == "mp":
+        if distributed_executor_backend == "ray":
```

Comment:
Instead of repeating this logic in both `LLMEngine` and `AsyncLLM`, can we instead move `_get_executor_cls` into abstract `Executor`? That way, we don't need to repeat logic across implementations. This is not a `@classmethod` anyways

Reply:
Good catch. Yeah we should unify them. cc @ruisearch42